e2e/consul/input/on_update_check_restart.nomad

```diff
@@ -48,7 +48,6 @@ job "test" {
         on_update = "ignore_warnings"
 
         args = [
-          "-c",
```

Comment:
This was causing the wrong error code because `-c` means "run this script literal", not "run this script file".